lib/components/table.vue

```diff
@@ -17,7 +17,8 @@
                     :tabindex="field.sortable?'0':null"
                 >
                   <slot :name="'HEAD_'+key" :label="field.label" :column="key" :field="field">
```

Comment:
This is a named slot.  you can reference the slot by name to place content in any of the header cells, which will override the default content inside this slot.

See https://vuejs.org/v2/guide/components.html#Scoped-Slots